zulipterminal/ui_tools/views.py

```diff
@@ -807,8 +807,12 @@ def __init__(self, controller: Any,
 
 
 class AboutView(PopUpView):
-    def __init__(self, controller: Any, ZT_VERSION: str) -> None:
-        content = [
+    def __init__(self, controller: Any, ZT_VERSION: str, server_version: str,
+                 server_feature_level: str) -> None:
+        content = ([('Server feature level', server_feature_level)]
+                   if server_feature_level else [])
+        content += [
+            ('Server version', server_version),
             ('Zulip Terminal version', ZT_VERSION),
```

Comment:
Would it be clearer to put the ZT version first?

If we move to categories based on the other PR, we could group them instead?